schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -550,6 +571,130 @@
         "InstallerUrl",
         "InstallerSha256"
       ]
+    },
+    "NestedInstaller": {

```

Comment:
Instead of duplicating everything from the installer, why not just reference the installer entry and extend it? It's possible to negate the InstallerUrl and the InstallerType
```json
    "NestedInstaller": {
      "type": "object",
      "properties": {
        "allOf": {
          "$ref": "#/definitions/Installer"
        },
        "InstallerUrl": {
          "not": {}
        },
        "InstallerType": {
          "not": {}
        },
        "NestedInstallerType": {
          "type": [
            "string",
            "null"
          ],
          "enum": [
            "msix",
            "msi",
            "appx",
            "exe",
            "inno",
            "nullsoft",
            "wix",
            "burn",
            "portable"
          ],
          "description": "Enumeration of supported nested installer types that could be contained in a zip or archive file."
        },
        "RelativeFilePath": {
          "type": [
            "string",
            "null"
          ],
          "description": "Relative path to the installer file or portable executable. Only applies to the nested installer within a Zip. "
        }
      },
      "required": [
        "Architecture",
        "InstallerSha256",
        "NestedInstallerType"
      ]
    }
```

Reply:
Thank you for the suggestion. Updated with suggested changes. Looks like a lot cleaner now :)